db/db.go

```diff
@@ -0,0 +1,23 @@
+package db
+
+import (
+	"database/sql"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+const create string = `create table users(id integer primary key autoincrement, username text, password text, timeslots blob);`
+
+var Db_instance *sql.DB
```

Comment:
use proper naming, please. In go we use Camel/Pascal case, and mostly avoid snake case.